core/ui_mixins/quick_panel.py

```diff
@@ -168,7 +162,7 @@ def __init__(self, items, on_done, **kwargs):
         self.on_done = on_done
         for option in ['flags', 'selected_index', 'on_highlight',
                        'limit', 'next_message', ]:
-            if option in kwargs:
+            if option in kwargs and kwargs[option]:
```

Comment:
why add `and kwargs[option]` ? won't this effectively skip any "falsy" value?

E.g doing: `PaginatedPanel(selected_index=0, limit=0)` would ignore both values and keep them at the default `None`.